sympy/series/limits.py

```diff
@@ -329,6 +327,31 @@ def remove_abs(expr):
                 r = gruntz(e, z, z0, dir)
             if r is S.NaN or l is S.NaN:
                 raise PoleError()
+        except SignException as er:
+            sig = er.sign
+            r = None
```

Comment:
limit() should not return None. If the limit can't be computed it should either raise an exception or return a `Limit` object.